scripts/DR_distributions_cross_run_analysis.py

```diff
@@ -0,0 +1,228 @@
+USAGE = """
+
+Summarize deed-restricted units by source, year-built, and locations.
+
+"""
+
+import glob, os, time
+import pandas as pd
+import numpy as np
+
+
+# set up file directory and run_id dictionary
+# when we have new runs, copy the 'building_data_2050.csv' output to this directory,
+# and add the new run_id and BAUS version to runid dictionry 
+
+run_dir = 'M:\\Data\\Urban\\BAUS\\PBA50\\Final_Blueprint\\run_analysis\\DR analysis'
+os.chdir(run_dir)
+
+# run_id dictionary
+runid = {'run325': 'v2.2.1',
+         'run241': 'v2.3',
+         'run242': 'v2.3.1',
+         'run152': 'v2.4',
+         'run327': 'v2.5',
+         'run243': 'v2.6',
+         'run158': 'v2.7',
+         'run335': 'v2.8',
+         #'run160': 'v2.9',
+         #'run253': 'v2.10',
+         #'run161': 'v2.11',
+         #'run340': 'v2.12',
+         'run254': 'v2.13',
+         'run171': 'v2.14',
+         'run347': 'v2.15',
+         'run261': 'v2.16'}
+
+
+if __name__ == '__main__':
+
+
+    ###### step 1: get parcel_id - geographies mapping
+
+    p_geo = pd.read_csv('util\\2020_09_21_parcels_geography.csv',
+                        usecols = ['PARCEL_ID', 'juris',
+                                   #'fbpzoningm', 'fbp_gg_id', 'pda_id_pba50_fb', 'fbp_tra_id', 
+                                   'fbp_sesit_id'])
+    juris_county = pd.read_csv('util\\juris_county_id.csv', 
+                               usecols = ['juris_name_full', 'baus_output_juris_name', 'county_name'])
+
+    p_taz = pd.read_csv('util\\2020_08_17_parcel_to_taz1454sub.csv',
+                        usecols = ['PARCEL_ID', 'ZONE_ID'],
+                        dtype = {'PARCEL_ID': int, 'ZONE_ID': int})
+
+    taz_sd = pd.read_csv('util\\taz_geography.csv', usecols = ['zone','superdistrict'])
+    taz_sd.columns=['zone','sd_number']
+
+    sd = pd.read_csv('util\\superdistricts.csv', usecols = ['number', 'name', 'subregion'])
+    sd.columns = ['sd_number','superdistrict','subregion']
+
+    p_geo = p_geo.merge(juris_county,
+                        left_on='juris',
+                        right_on='juris_name_full',
+                        how='left').merge(p_taz,
+                                          on='PARCEL_ID',
+                                          how='left').merge(taz_sd,
+                                                            left_on='ZONE_ID',
+                                                            right_on='zone',
+                                                            how='left').merge(sd,
+                                                                              on='sd_number',
+                                                                              how='left')
+    print(p_geo.head())
+
```

Comment:
Could we comment here what the columns are in p_geo at this point?

Reply:
Not a request, just notes: p_geo has columns: 
* 'PARCEL_ID', 'juris', 'fbp_sesit_id'
* 'juris_name_full', 'baus_output_juris_name', 'county_name' (from juris_county)
* 'zone_id' (from p_taz)
* 'sd_number','superdistrict','subregion' from sd